index.html

```diff
@@ -14,7 +14,7 @@
 
   gtag('config', 'G-1NWRQ1EMTE');
```

Comment:
Please add the accessibility report in Pull Request.

The report should include a check about page titles.
The report should include a check about image text alternatives (alt="alt text ").
The report should include a check about text headings.
The report should include a check about color contrast.
The report should include a check about resize.
The report should include a check about interaction.
The report should include a check about moving content.
The report should incldue a check about multimedia.
The report should include a check about the basic structure of the page.

If everything is ok, please make a statement about that (e.g. "Page titles - no issues found").
If issues are found, please add  link to the corresponding line of code in the Pull Request changes